spring-cloud-netflix-eureka-client/src/test/java/org/springframework/cloud/netflix/eureka/EurekaClientAutoConfigurationTests.java

```diff
@@ -215,17 +216,16 @@ public void statusPageUrl_and_healthCheckUrl_do_not_contain_server_context_path(
 	}
 
 	@Test
-	public void statusPageUrl_and_healthCheckUrl_contain_management_context_path() throws Exception {
+	public void statusPageUrl_and_healthCheckUrl_are_not_set_when_management_port_is_not_set() throws Exception {
 		addEnvironment(this.context,
 				"server.port=8989", "management.server.context-path=/management");
 
 		setupContext(RefreshAutoConfiguration.class);
 		EurekaInstanceConfigBean instance = this.context
 				.getBean(EurekaInstanceConfigBean.class);
-		assertTrue("Wrong status page: " + instance.getStatusPageUrl(),
-				instance.getStatusPageUrl().endsWith(":8989/management/info"));
-		assertTrue("Wrong health check: " + instance.getHealthCheckUrl(),
-				instance.getHealthCheckUrl().endsWith(":8989/management/health"));
+
+		assertThat(instance.getStatusPageUrl()).isNull();
```

Comment:
I'm confused why these aren't set? Just because the port isn't set, doesn't mean status and health check urls don't exist.

Reply:
It's not like they don't exist at all, they only don't get overriden here. They later get built by `InstanceInfo.Builder` upon registration (please see the Builder's setters).

This change only fixes broken communication with Turbine brought by M5, by restoring previous behaviour for the managementPort-not-set case. If other behaviour is needed I think it's a separate issue.